packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -84,4 +88,10 @@
     margin-left: var(--terra-menu-item-chevron-icon-margin-left, 10px);
     margin-right: var(--terra-menu-item-chevron-icon-margin-right, 10px);
   }
+
+  .instructionsforuse {
```

Comment:
Should this be themeable?

Reply:
I am not sure whether this Icon needs to themeable or not like other icons of menu-item. Need UX Input @neilpfeiffer

Reply:
@nramamurth @supreethmr -- This should **not** be themeable - we are required to use the same icon for all cases. 
You will want to expose the text `color` as a variable property so it can be re-colored as appropriate in other themes like lowlight/dark.